### DISCUSSION TITLE
Ignore Model Store Persistence ?

### DISCUSSION THREAD

**Discussion by RadouaneEzzaggab** [infinitered/ignite#1815] [Q&A] [answered]
Hello guys, sorry for my bad english.

Pls, How can i ignore one of my model store from data storage ?

**Comment by jamonholmgren** [comment, marked as answer]
Hey @RadouaneEzzaggab, thanks for the question!

You can ignore sections by editing this:

https://github.com/infinitered/ignite/blob/d900ef552b87050f5df71c3f732462bb05975f4b/boilerplate/app/models/root-store/setup-root-store.ts#L51-L52

Something like this (not tested):

```tsx
  // track changes & save to storage
  onSnapshot(rootStore, (snapshot) => {
    // remove `someModel` from stored snapshot
    const { someModel, ...newSnapshot } = snapshot
    storage.save(ROOT_STATE_STORAGE_KEY, newSnapshot)
  })
```

**Reply by RadouaneEzzaggab** [reply]
Hey @jamonholmgren, thanks for the answer!

I think it is logical, i'll test it later.

**Reply by RadouaneEzzaggab** [reply]
Hey @jamonholmgren, i just test your code, it is work properly just like that: 

onSnapshot(rootStore, (snapshot) => {
    //  someModel will be stored, the rest won't be
    const { someModel, ...rest } = snapshot
    storage.save(ROOT_STATE_STORAGE_KEY, { someModel, ...rootStore })
})
